recipes/riboseq-rust/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "riboseq-rust" %}
+{% set version = "1.2" %}
+{% set sha256 = "2b50915fda23f9bc9c8170d82fa5255ef2c3954f0d6a5ea15649b119ff506d0c" %} 
+
+package:
+  name: riboseq-rust
+  version: "1.2"
+
+source:
+  fn: RUST_{{ version }}.tar.gz
+  url: https://lapti.ucc.ie/rust/scripts/RUST_{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [not py27 or osx]
+
+requirements:
+  build:
+    - python
+    - argparse
+    - matplotlib
+    - numpy
+    - pysam
+  run:
+    - python
+    - argparse
+    - matplotlib
+    - numpy
+    - pysam
+
+test:
+  commands:
+    - rust_nucleotide -h 
+about:
+  home: https://lapti.ucc.ie/rust/
+  license: GPL
```

Comment:
Can you add the license file and clarify the license with
```yaml
  license: GPL-3.0
  license_file: COPYING
```

Reply:
Updated the the license information according to your review.
Thanks @mbargull !